docs/migration/migrate_6_0_index.asciidoc

```diff
@@ -0,0 +1,337 @@
+[[migrating-6.0-index]]
+== Migrating Kibana index to 6.0
+
+[IMPORTANT]
+==============================================
+The following will make transformations to your Kibana index. It is recommended you backup your Kibana index prior to running any of the following commands.
+==============================================
+
+In order to migrate to Kibana 6, your Kibana index needs to be re-indexed. This will make transformations to the data model, removing types which are being {es-ref}/removal-of-types.html[removed in Elasticsearch].
+
+A UI is available in X-Pack 5.6 to assist with the migration. The following is provided should you be required to run the migration manually. The commands assume your using the default `.kibana` index. If `kibana.index` is set to something different in your `kibana.yml` you will need to modify the commands.
+
+To preform this migration without downtime, you will need to be running Kibana 5.6. This is the only version which supports both of the index types.
```

Comment:
not saying we want to go down this path but a no downtime upgrade without 5.6 could be to stand up two kibana's, one to the reindexed .kibana and update A records to the new IP after